ui/DesignSystem/Component/SourceBrowser/CommitTeaser.svelte

```diff
@@ -1,7 +1,11 @@
 <script>
+  import { getContext } from "svelte";
   import { link } from "svelte-spa-router";
   import { Text } from "../../Primitive";
   import UserCard from "../UserCard.svelte";
+  import * as path from "../../../lib/path.js";
+
+  const projectId = getContext("projectId");
```

Comment:
Why did you decide to pass this through the context instead of a component prop? Where is the context set?

Reply:
I agree, we should pass it in via a prop and the callers should be adjusted accordingly.